singletons/utils.js

```diff
@@ -317,7 +317,7 @@ module.exports = (function (Module) {
 					seconds -= (hr * Utils.timeUnits.h.s);
 				}
 				const min = Math.floor(seconds / Utils.timeUnits.m.s);
-				stuff.push(min);
+				stuff.push(stuff.length ? this.zf(min, 2) : min);
```

Comment:
wrap the ternary operator condition in parentheses

Reply:
Fixed in f2d66c5